components/gateway/internal/gqlschema/schema.graphql

```diff
@@ -395,8 +419,13 @@ type Mutation {
     deleteAPI(id: ID!): APIDefinition
     refetchAPISpec(apiID: ID!): APISpec
 
-    setAPICredential(apiID: ID!, in: CredentialInput!): Credential!
-    deleteAPICredential(apiID: ID!): Credential
+    """
+    If runtimeID is not provided, the same authentication details will be set for every runtime already available.
+    Returns information for which runtime it was configured.
+    """
+    setAPIAuth(apiID: ID!, runtimeID: ID, in: AuthInput!): [RuntimeAuth!]!
```

Comment:
If there would be a default `Auth` like I suggested in one of my previous comments, providing no `runtimeID` here would set the mentioned default `Auth`.